core/src/main/java/org/openmbee/sdvc/core/config/ContextObject.java

```diff
@@ -70,6 +70,6 @@ private void projectIdUpdated() {
     }
 
     private void branchIdUpdated() {
-        this.dbTableSuffix = this.branchId.equals(MASTER_BRANCH) ? "" : this.branchId.replace('-', '_');
+        this.dbTableSuffix = this.branchId.equals(MASTER_BRANCH) ? "" : this.branchId;
```

Comment:
this probably don't need to change with the quotes added now, would need to be tested..